src/JsonSchema/Constraints/UndefinedConstraint.php

```diff
@@ -253,8 +253,14 @@ protected function applyDefaultValues(&$value, $schema, $path)
                 }
             }
         } elseif (isset($schema->items) && LooseTypeCheck::isArray($value)) {
+            $items = array();
+            if (LooseTypeCheck::isArray($schema->items)) {
+                $items = $schema->items;
+            } elseif (isset($schema->minItems) && count($value) < $schema->minItems) {
+                $items = array_fill(count($value), $schema->minItems - 1, $schema->items);
```

Comment:
Why the magic number? Shouldn't this be less `count($value)`, rather than less `1`?